src/Paysera/Component/Console/PharKernel.php

```diff
@@ -1,31 +0,0 @@
-<?php
-
-namespace Paysera\Component\Console;
-
-use AppKernel;
-
-class PharKernel extends AppKernel
```

Comment:
why this class is removed?

Reply:
I think this was added, because the regular kernel tries to build the cache if there isn't any, which will be problematic for phar files. I think we can build the cache before creating the phar file and include the cache to the payload instead.